### PR TITLE
Adding hersey.hackclub.com

### PR DESCRIPTION
**Pull request by AAlchi**
# [Adding] `hersey.hackclub.com`

## Description

This subdomain will be used for our school's Hack Club (John Hersey High School). Eventually, it will be updated around every week to have Hack Club members go to the site.

### CODE REVIEW

File: hackclub.com.yaml [hunk 1]
```diff
@@ -1473,6 +1473,10 @@ help.bank:
 help.hcb:
   type: CNAME
   value: hcb.helpscoutdocs.com.
+hersey:
+  - ttl: 600
+    type: CNAME
+    value: hersey-hack-club.vercel.app/.
```
Comment (Muirrum) [inline hackclub.com.yaml:1479]: DNS records can't have paths `/`


```suggestion
    value: hersey-hack-club.vercel.app.
```

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml [hunk 2]
```diff
@@ -1473,6 +1473,10 @@ help.bank:
 help.hcb:
   type: CNAME
   value: hcb.helpscoutdocs.com.
+hersey:
+  - ttl: 600
+    type: CNAME
+    value: hersey-hack-club.vercel.app.
```
Comment (Muirrum) [inline hackclub.com.yaml:1479]: iirc vercel's dns instructions are:

```suggestion
    value: cname.vercel-dns.com.
```

**Review by Muirrum — CHANGES_REQUESTED** [review]
please also complete subdomain verification per vercel's instructions

**Review by AAlchi — COMMENTED** [review]
Hello. I have made the requested changes and I don't want to rush, but I'd just like to know when to expect this to be ready. I'm good for this week, but I want to make sure that the website is up by next week. Thank you!

**Comment by AAlchi** [comment]
How do I add the vercel verification record?

**Comment by Muirrum** [comment]
Is this still needed

**Comment by AAlchi** [comment]
I'm confused on how it's supposed to work and I'm not really sure what to do. Can you please explain it in more detail?

**Comment by Muirrum** [comment]
https://github.com/hackclub/dns?tab=readme-ov-file#limitations

Your vercel verification needs to go under the root _vercel record at the top of the file.

**Comment by AAlchi** [comment]
I added the change. Please lmk if there is anything else I need to do.

**Review by alx-alexpark — APPROVED** [review]
yum

**Pull request merged**